snuba/cogs/accountant.py

```diff
@@ -0,0 +1,250 @@
+from __future__ import annotations
```

Comment:
The library is now public. You do not need to vendor in the accumulator.
https://pypi.devinfra.sentry.io/simple/sentry-usage-accountant/index.html